compiler/modules/bir/bir.bal

```diff
@@ -592,21 +592,22 @@ public type EqualityInsn readonly & record {|
 # any function call could result in a stack overflow panic.
 # XXX This does not handle functions that don't return
 # (i.e. with return type of never)
+# XXX: This can also panic due to memory allocation for uniform function call
+# which is not handled gracefully
 public type CallInsn readonly & record {|
```

Comment:
Please add a comment describing the restriction on the func SemType

Reply:
Fixed (add a comment with `restArgIsList`)